packages/@vue/cli/lib/util/generateReadme.js

```diff
@@ -0,0 +1,26 @@
+const descriptions = {
+  build: 'Compiles and minifies for production',
+  serve: 'Compiles and hot-reloads for development',
+  lint: 'Lints and fixes files',
+  test: 'Run your tests',
+  'test:e2e': 'Run your end-to-end tests',
+  'test:unit': 'Run your unit tests'
+}
+
+function printScripts (pkg, packageManager) {
+  return Object.keys(pkg.scripts).map(key => {
+    return [
+      `\n### ${descriptions[key]}`,
```

Comment:
about `descriptions`, I think it should be decided dynamically according to features selection at `vue create`.

In that implementation, the description which is not necessary is generated in the `README.md`.
for example, in the features selection, even if unit test feature is not selected, it will be output to `README.md`.

Reply:
But the file is generated based on the keys on the `scripts` section of `package.json`. If you don't select unit testing, it won't output the description on `README.md`.

Reply:
That's true.

In that phase, it's works, due to `package.json` contents is resolved with `Generator`.
For #1675, There seems to be no problem with implementation.  ✨ 

As the concern, If third party plugin author want to  extend `README.md` dynamically, I think that it's not that simple implementation, but it need to implement at `Generator` or `cli-service`.

Reply:
@kazupon That would be even better